docs/onboarding/01-project-overview.md

```diff
@@ -0,0 +1,92 @@
+# Yellowstone Vixen - Project Overview
+
+## Introduction
+Yellowstone Vixen is a high-performance Rust-based project that provides a gRPC service for parsing Solana transactions. It's designed to work seamlessly with the Yellowstone ecosystem, offering efficient and reliable transaction parsing capabilities.
```

Comment:
It doesn't provide gRPC service but uses gRPC stream of change event updates. Vixen is framework for creating and apply parsers to decode the solana change events.

Reply:
Thanks for the clarification, let me update this! <3